src/axom/core/numerics/matvecops.hpp

```diff
@@ -571,9 +575,11 @@ inline bool linspace( const T& x0, const T& x1, T* v, int N )
 template < typename T >
 inline void cross_product( const T* u, const T* v, T* w )
 {
-  assert( "pre: u pointer is null" && (u != nullptr) );
-  assert( "pre: v pointer is null" && (v != nullptr) );
-  assert( "pre: w pointer is null" && (w != nullptr) );
+  #ifndef AXOM_USE_RAJA
+    assert( "pre: u pointer is null" && (u != nullptr) );
```

Comment:
CUDA has a native `assert()` -- this shouldn't be ifdefed out.

Reply:
I had these `assert()` guarded because of a compiler bug I found with CUDA 10.2 on rzansel. I have not ran into the issue with CUDA 10.1, so it should be fine to leave them unguarded.